src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -354,21 +355,24 @@ public string GetHelpMaml(string helpItemName, string path)
         protected override ProviderInfo Start(ProviderInfo providerInfo)
         {
             // Set the home folder for the user
-            if (providerInfo != null && string.IsNullOrEmpty(providerInfo.Home))
+            if (providerInfo != null)
```

Comment:
I think you can revert changes in this file.

Reply:
Done